docs/pages/blog/hello-grid-version-2.md

```diff
@@ -0,0 +1,144 @@
+
+```js
+{
+  // --Grid-columns is defined in the grid container
+  width: `calc(100% * ${value} / var(--Grid-columns))`,
+}
+```
+
+Another example is how the padding is calculated in grid items:
+
+In `Grid v1`, grid items has no information about the padding it should have because the `spacing` prop is defined on the grid container. So, the simplest way is to control the padding from the grid container using CSS child combinator (`>`):
+
+```js
+function getOffset(val) {
+  const parse = parseFloat(val);
+  return `${parse}${String(val).replace(String(parse), '') || 'px'}`;
+}
+// ...The code is shorten for readability
+({
+  [`& > .${gridClasses.item}`]: {
+    // Grid item's padding is controlled by its container
+    paddingTop: getOffset(themeSpacing),
+  },
+});
+```
+
+However, in `Grid v2`, the javascript logic can be separated independently because CSS variables shift the communication between container and item to CSS.
+
+- **Grid container** takes care of the variables generation based on the `spacing` prop:
+  ```js
+  // The code is simplified for readability
+  {
+    '--Grid-rowSpacing': spacingProp,
+    '--Grid-columnSpacing': spacingProp,
+  }
+  ```
+- **Grid item** refers to the variables:
+  ```js
+  {
+    padding: `calc(var(--Grid-rowSpacing) / 2) calc(var(--Grid-columnSpacing) / 2)`,
+  }
+  ```
+
+This is a very good use case for applying CSS variables when it comes to components that has a parent-child relationship which also support media query customization.
+
+#### Remove unnecessary props
+
+In version 2, `Grid` is always an item which is similar to the flexbox item in CSS, so the `item` prop is not needed anymore.
+
+We also update the grid styles to work with ellipsis text by default, so the `zeroMinWidth` prop is removed.
+
+```diff
+ <Grid container spacing={2}>
+-  <Grid item zeroMinWidth><Typography>Long text...</Typography></Grid>
++  <Grid><Typography>Long text...</Typography></Grid>
+ </Grid>
+```
+
+#### Nested grid
+
+With CSS variables and the removal of the `item` prop, there is no more limitation for creating nested grids.
+
+As a bonus, a grid container automatically inherits row and column spacing values from the root grid container unless they are specified directly on the component.
+
+```js
+import Grid from '@mui/material/Unstable_Grid2`;
+
+// root grid container
+<Grid container spacing={2}>
+  <Grid>...</Grid>
+  <Grid container> {/* inherits spacing from the root container */}
+    <Grid>...</Grid>
+    <Grid>...</Grid>
+  </Grid>
+  <Grid>...</Grid>
+</Grid>
+```
+
+## Future plan and migration
+
+Since the `Grid v2` is rewritten from scratch, it is currently considered Unstable\_ as we give the community time to try it out and offer feedback. We will make it stable and deprecate v1 in the next major release of Material UI.
```

Comment:
```suggestion
Since the `Grid` has been rewritten from scratch for v2, it is currently considered _unstable_ as we give the community time to try it out and offer feedback. 
We will make it stable and deprecate v1 in the next major release of Material UI.
```